app-text/multimarkdown/multimarkdown-6.4.0.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Lightweight markup processor to produce HTML, LaTeX, and more"
+DESCRIPTION="MultiMarkdown, or MMD, is a tool to help turn minimally marked-up \
+plain text into well formatted documents, including HTML, PDF (by way of LaTeX), \
+OPML, or OpenDocument (specifically, Flat OpenDocument or ‘.fodt’, which can \
+in turn be converted into RTF, Microsoft Word, or virtually any other \
+word-processing format)."
+HOMEPAGE="https://fletcherpenney.net/multimarkdown/"
+COPYRIGHT="2016-2018 Fletcher T. Penney
+	2005-2016 Troy D. Hanson
+	2013-2014 RAD Game Tools and Valve Software
+	2010-2014 Rich Geldreich and Tenacious Software LLC
+	1998-2001,2003-2011,2013 Stewart Heitmann"
+LICENSE="BSD (3-clause)
+	MIT
+	Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/fletcher/MultiMarkdown-6/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="fa9daa089bc0f8bc1f69fd7365778d59210b446ce7008d03b87fb572df8ff5d3"
+SOURCE_FILENAME="multimarkdown-$portVersion.tar.gz"
+SOURCE_DIR="MultiMarkdown-6-$portVersion"
+PATCHES="multimarkdown-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
test could not run because perl is not found while testing even if I add TEST_REQUIRES="cmd:perl".
I've also fixed shebung to /bin/perl in MarkdownTest.pl but gives:

    runtime_loader: Cannot open file /bin/perl: No such file or directory

Reply:
You can try this shebang instead:

```sh
#!/usr/bin/env perl
```

Reply:
I've tried #!/bin/env perl, but env failed to find perl. I tried to pass the path to perl to ctest by hacking add_test calls in build/CTestTestfile.cmake, but ctest failed to find perl.

Reply:
It looks the problem happen because of --no-repository-update flag for haikuporter which I used sometimes. Now test works well.